.github/actions/eks-cleanup-resources/README.md

```diff
@@ -0,0 +1,39 @@
+# Delete EKS Resources
+
+This GitHub Action automates the deletion of AWS resources using a shell script. It helps you manage and clean up modules of this repository as resources by specifying a target or deleting resources based on age criteria.
+
+## Usage
+
+To use this action, include it in your workflow file (e.g., `.github/workflows/delete-eks-resources.yml`):
+
+```yaml
+name: Delete EKS Resources
+
+on:
+  workflow_dispatch:
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete EKS resources
+        uses: camunda/camunda-tf-eks-module/eks-cleanup-resources@main
+        with:
+          tf-bucket: 'your-s3-bucket-name'
+          tf-bucket-region: 'your-region'
+          max-age-hours: 24
+          target: 'all'
+          temp-dir: './tmp/eks-cleanup/'
+```
+
+## Inputs
+
+The action supports the following input parameters:
+
+| Input Name         | Description                                                                               | Required | Default                    |
+|--------------------|-------------------------------------------------------------------------------------------|----------|----------------------------|
+| `tf-bucket`        | The S3 bucket containing the resources' state files.                                       | Yes      | N/A                        |
+| `tf-bucket-region` | The region of the S3 bucket containing the resources state files. Falls back to `AWS_REGION` if not set. | No       | AWS_REGION                 |
+| `max-age-hours`    | The maximum age (in hours) for resources to be deleted.                                    | No       | "20"                       |
+| `target`           | Specifies an ID to destroy specific resources or "all" to destroy all resources.           | Yes      | "all"                      |
```

Comment:
Welcome to tonights game show, is it required or not.
![](https://media1.tenor.com/m/NNp_ehNBMEoAAAAC/excited-so.gif)

Tonights guest is @leiicamundi and the famous question comes, is it a required input if it has a default.
![](https://media1.tenor.com/m/OlLumtC4ZnwAAAAC/sale-of.gif)

Tune in tonight, to find the answer!
![](https://media1.tenor.com/m/vxQ2ZlN7yScAAAAd/game-gameshow.gif)

Reply:
but Terraform isn't HTML and doesn't have the same logic.
I understand what you mean, but in the end, I can leave it empty in Terraform since we have a default, so it isn't required.